lib/puppet/provider/consul_acl/default.rb

```diff
@@ -53,18 +46,11 @@ def self.list_resources
     nacls
   end
 
-  def self.prefetch(resources)
```

Comment:
I've never written a custom provider, but it does seem better to keep the prefetch.  Maybe there's a way to configure a provider globally?